src/features/pass-survey/model/hooks/useActivityStepper.ts

```diff
@@ -36,12 +37,16 @@ function useActivityStepper(state: ActivityState | undefined) {
 
   const canSkip =
     !!currentPipelineItem?.isSkippable && !hasAnswer && !isSplashStep;
-  const canMoveNext =
+
+  const canContinue =
     isTutorialStep ||
     isMessageStep ||
     isAbTestStep ||
     currentPipelineItem?.isSkippable ||
     (hasAnswer && (!additionalAnswerRequired || hasAdditionalAnswer));
+
+  const canMoveNext = canContinue && answerValidator.isValidAnswer();
```

Comment:
I just realized that we should bypass items if the items are skippable. Please move the validation to the 46 line:
`(hasAnswer && (!additionalAnswerRequired || hasAdditionalAnswer)) && answerValidator.isValidAnswer();`